requirements.txt

```diff
@@ -6,7 +6,7 @@ jupyter>=1.0.0
 langchain
 langchain-community
 langchain-experimental
-modelscope>=1.9.3
+modelscope==1.11.0
```

Comment:
should have a loose constrain on modelscope version

Reply:
The requirements for the modelscope version have been updated to a runnable range